mass_editing/models/mass_editing_line.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# Copyright (C) 2020 Iván Todorovich (https://twitter.com/ivantodorovich)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MassEditingLine(models.Model):
+    _name = "mass.editing.line"
+    _description = "Mass Editing Line"
+    _order = "sequence, field_id"
+
+    sequence = fields.Integer()
+    server_action_id = fields.Many2one(
+        "ir.actions.server",
+        string="Server Action",
+        ondelete="cascade",
+        required=True,
+    )
+    model_id = fields.Many2one(
+        "ir.model",
+        related="server_action_id.model_id",
+    )
+    field_id = fields.Many2one(
+        "ir.model.fields",
+        string="Field",
+        domain="""
+            [
+                ("name", "!=", "__last_update"),
+                ("ttype", "not in", ["reference", "function"]),
+                ("model_id", "=", model_id),
+            ]
+        """,
+        ondelete="cascade",
+        required=True,
+    )
+    widget_option = fields.Char(
+        string="Widget Option",
+        help="Add widget text that will be used to display the field in the wizard.\n"
+        "Example: 'many2many_tags', 'selection', 'image'",
+        compute="_compute_widget_option",
+        readonly=False,
+        store=True,
+    )
+    apply_domain = fields.Boolean(
+        default=False,
+        string="Apply domain",
+        help="Apply default domain related to field",
+    )
+
+    @api.depends("field_id")
+    def _compute_widget_option(self):
+        for rec in self:
+            if rec.field_id.ttype == "many2many":
+                rec.widget_option = "many2many_tags"
+            elif rec.field_id.ttype == "binary":
```

Comment:
Maybe a little optimistic, dont you think ? Or you could add test if name contains image or logo. You should handle 99% of the images. Dont you think ?

Reply:
My bad. Somehow I missed that line when I rewrote this block :sweat_smile: Thanks!!
It's fixed now